randovania/games/am2r/gui/dialog/cosmetic_patches_dialog.py

```diff
@@ -11,10 +14,28 @@
 
     from randovania.layout.base.cosmetic_patches import BaseCosmeticPatches
 
+DEFAULT_HEALTH_COLOR = (255, 225, 0)
+DEFAULT_ETANK_COLOR = (112, 222, 250)
+DEFAULT_DNA_COLOR = (46, 208, 5)
+
+def hue_rotate_color(original_color: tuple[int, int, int], rotation: int):
+    color = QColor.fromRgb(*original_color)
+    h = color.hue() + rotation
+    s = color.saturation()
+    v = color.value()
+    while h >= 360:
+        h -= 360
+    while h < 0:
+        h += 360
+
+    rotated_color = QColor.fromHsv(h, s, v)
+    return (rotated_color.red(), rotated_color.green(), rotated_color.blue())
```

Comment:
We should maybe move this function somewhere that prime and am2r can access it since both implementations are identical

Reply:
Can I just be lazy for now and mark that as a `TODO`?
Also not sure where the location should be. I thought about `randovania/lib/color_lib.py` but making a whole class for one function seems a bit overkill.

Reply:
I'm also fine with adding a `TODO`.

The helper function could be added later to `base_cosmetic_patches_dialog.py` as both are importing that module anyways or just to a new file in `gui\lib` as there are already other modules with only one helper function like `wait_dialog.py` or `error_message_box.py`